### ISSUE TITLE
nnU-Net Custom Training: Feature Request

### ISSUE THREAD

**Issue by sbmalik** [NVIDIA/DeepLearningExamples#1115]
Update Custom Training Notebook for nnU-Net

*Examples:* 
* *PyTorch* 
* *Jupyter notebooks*

**Is your feature request related to a problem? Please describe.**
Current notebook for custom training not working. 

**Describe the solution you'd like**
An updated version of custom 2d dataset training notebook

**Additional context**
some imports cause errors, but I resolved them. After that DataLoader for validation data is giving following error
`update() takes 3 positional arguments but 4 were given`

**Comment by sbmalik** [comment]
@michal2409 can you please look into this.. ?

**Comment by michal2409** [comment]
Hi @sbmalik, do you run the notebook on xView2 dataset or using something different? In case you are using different dataset you might to modify some code, e.g. the metric update function.

**Comment by sbmalik** [comment]
@michal2409 thank you for response, I tried with the same dataset but getting the error because of validation dataloader (I think), mentioned in question. Moreover you can reproduce the error by just running the notebook.
